secretsauce/apps/portal/models.py

```diff
@@ -48,7 +50,7 @@ class DataBlock(models.Model):
         related_name='data_blocks'
     )
     name = models.CharField(max_length=200, unique=True)
-    upload = models.FileField(upload_to='uploads/')
+    upload = models.FileField(upload_to=obfuscate_upload_link)
 
```

Comment:
As mentioned, can we add a type field to this? To indicate whether it is training dataset or costdataset

Reply:
Cost dataset should be uploaded using /portal/project/<uuid:pk>/items/, they are not the same as DataBlock. They have different headers, etc.